source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container.ts

```diff
@@ -16,23 +26,26 @@ abstract class VivaConnectionsContainer extends HostContainer {
         hostElement.appendChild(vcContainer);
     }
 
+    public getHostConfig(): Adaptive.HostConfig {
+        return new Adaptive.HostConfig(
+            this._colorTheme === ColorTheme.Light ? hostConfigLight : hostConfigDark
+        );
+    }
+
+    public getBackgroundColor(): string {
+        return this._colorTheme === ColorTheme.Light ? "#F6F6F6" : "#1b1a19";
+    }
+
     get targetVersion(): Adaptive.Version {
         return Adaptive.Versions.v1_3;
     }
-}
-
-export class LightVivaConnectionsContainer extends VivaConnectionsContainer {
-	public getHostConfig(): Adaptive.HostConfig {
-			return new Adaptive.HostConfig(hostConfigLight);
-	}
-}
 
-export class DarkVivaConnectionsContainer extends VivaConnectionsContainer {
-	public getBackgroundColor(): string {
-			return "#1b1a19";
-	}
+    set colorTheme(value: ColorTheme) {
+        this._colorTheme = value;
+        this.styleSheet = `containers/viva-connections-container-${this._colorTheme.toLowerCase()}.css`;
```

Comment:
You can define the rule and implement it in the base class.

for example:
this.styleSheet = 'containers/' + container + `-${this._colorTheme.toLowerCase()}.css`